samples/e2e-mnist/mnist.ipynb

```diff
@@ -13,9 +13,11 @@
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "### Define the model name and Kubeflow user namespace\n",
+    "### Define the model name, Kubeflow user namespace, and storageclass for running this pipeline\n",
     "\n",
-    "Change the below cell with a name you want to use for this pipeline and which namespace you want to execute on Kubeflow."
+    "Change the below cell with a name you want to use for this pipeline, which namespace you want to execute on Kubeflow, and the [storageclass](https://kubernetes.io/docs/concepts/storage/storage-classes/) to run and store the trained model. **Make sure the storageclass below supports ReadWriteMany in order to run distributed training.** \n",
```

Comment:
Do you mind adding this as one prerequisite in `README.md`?

Reply:
done